GoogleChromeLabs/ps-analysis-tool
# Feature: Add blocking direction icons in cli

## Description
To increase feature parity between `extension` and the `CLI`, the blocking direction feature has been added to the CLI.

## Relevant Technical Choices
- Add network traffic details to the cookies.
- Derive the network blocking status from the network events.

## Testing Instructions
- Clone this branch. In the terminal run `npm run build-cli`.
- Now run `npm run cli https://livemint.com -- -nt`
- Post this open the HTML file generated by the CLI. In the table, you'll see the blocking direction signs.

## Screenshot/Screencast
![Screenshot 2024-06-17 at 14 16 40](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/59614577/461f6dad-5e4c-4727-b5b7-2a77138220b4)

---

## Checklist






Partially addresses #656
